packages/storybook/stories/va-breadcrumbs.stories.jsx

```diff
@@ -32,3 +83,6 @@ Default.args = {
   ...defaultArgs,
 };
 Default.argTypes = propStructure(breadcrumbsDocs);
+
+export const DynamicState = DynamicCrumbsTemplate.bind(null);
```

Comment:
Is "Dynamic State" the best way to describe this new Storybook story?

![Screenshot 2023-01-05 at 2 43 01 PM](https://user-images.githubusercontent.com/872479/210876235-e4d9d6e8-04b9-4c1b-88a9-4fc6cad39c2f.png)

Reply:
Also would it make sense to add additional documentation on the Storybook example to point out that the structure for dynamically updating crumbs need to be in the `<li><a href="...">...</a></li>` format?  (For example: "Note that the crumb structure needs to include a list item")

Or is the Storybook example clear enough?
 
![Screenshot 2023-01-06 at 10 33 47 AM](https://user-images.githubusercontent.com/872479/211055719-b65e8357-6321-4f70-9102-dc66a4ef2f4b.png)

Reply:
I would say... Add more info to make it clear that it's necessary to have it to make it work